sarahcalvis/giving-tree
# Integrated Stripe Setup with Foundation Dashboard

If a foundation gets to the Stripe Setup page, they will be directed through the process to the foundation dashboard. I did not handle directing the foundation to the Stripe Setup page if they do not have Stripe set up already because I think @Stahl4121 is going to handle that by checking if the foundation has a ```stripe_id``` in the database upon login but I could be wrong about that!

In order to test this:
1. log in as a foundation
2. go to foundation/stripe-setup
3. click "connect to stripe"
4. click "Skip this account form"
5. you should be redirected to the foundation dashboard and see a "connected to stripe" Snackbar.
6. you can check in the database to confirm that the Stripe ID Of your foundation truly has been updated. (It will be different from the code you see in the URL parameters because the code is used to get the Stripe ID.)
